webpack/assets/javascripts/react_app/redux/middlewares/index.js

```diff
@@ -15,6 +15,10 @@ const logReduxToConsole = () => {
   return isProduction && isLogger;
 };
 
+/**
+ * The order of middlewares is important,
+ * the action will be passed between middlwares in the order they are written.
+ * read more about it in https://redux.js.org/api/applymiddleware */
```

Comment:
Is this intentional?

Reply:
Yeah I wanted to trigger travis to work so we could compare gh actions with it.
Plus a nice to have comment :)